spec/system/work_spec.rb

```diff
@@ -39,16 +39,18 @@
   end
 
   it "Renders in wizard mode when requested", js: true do
-    work = FactoryBot.create(:draft_work)
+    draft_work = FactoryBot.create(:draft_work)
+    draft_work_submitter = User.find(draft_work.created_by_user_id)
```

Comment:
There is a method on the work `.created_by_user` that could be utilized here (Not a required change)